src/api.js

```diff
@@ -4,4 +4,8 @@ export default class API {
   static getMovies() {
     return Axios.get('http://localhost:3000/api/v1/movies');
   }
+
+  static getData() {
```

Comment:
Let's rename it to smth like searchFilms. And it should get to params which you pass: searchFor and searchBy

Reply:
need some explanations